docs/adr/0002-videos-languages.rst

```diff
@@ -0,0 +1,64 @@
+ADR 0002 - Videos languages
+===========================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+We want to think Marsha as accessible from the beginning. At least from the point of view of the videos, which are the main content available.
+
+We can think about a video as a "set of related videos with their accessibility counterparts".
+
+Counterparts
+~~~~~~~~~~~~
+
+Audio
++++++
+
+The same video can be proposed with different audio tracks. Which often result in many videos. That's our case, we don't manage audio tracks separately.
+
+Subtitles
++++++++++
+
+In addition to audio tracks, many subtitles tracks can be available.
+
+Sign language
++++++++++++++
+
+Some people with disabilities could want a video with the sign language transcript. For this it can be a video incorporated in the original one, or an other video displayed on the site.
+
+As sign languages are not the same for every spoken language, it can exist many sign languages videos for a single video.
```

Comment:
s/it can exist many sign languages videos/there can be several sign language videos ?